internal/init/init.go

```diff
@@ -132,18 +135,19 @@ func (c *Client) NewCommand() *cobra.Command {
 	return initCmd
 }
 
-func getFirstProjectID(s packngo.ProjectService, userOrg string) (string, error) {
-	listOpts := &packngo.ListOptions{}
-	listOpts.Including("organization")
-	listOpts.Excluding("devices", "members", "memberships", "invitations", "max_devices", "ssh_keys", "volumes", "backend_transfer_enabled", "updated_at", "customdata", "event_alert_configuration")
+func getFirstProjectID(s metal.ProjectsApiService, userOrg string) (string, error) {
+	include := []string{"organization"} // []string | Nested attributes to include. Included objects will return their full attributes. Attribute names can be dotted (up to 3 levels) to included deeply nested objects. (optional)
+	exclude := []string{"devices", "members", "memberships", "invitations", "ssh_keys", "volumes", "backend_transfer_enabled", "updated_at", "customdata", "event_alert_configuration"}
+	page := int32(56)    // int32 | Page to return (optional) (default to 1)
+	perPage := int32(56) // int32 | Items returned per page (optional) (default to 10)
```

Comment:
Overriding `perPage` is OK, but we'll still need to handle the pagination.  Here's an example of collecting paginated devices in the sweeper action; we'll need something like this for collecting projects: https://github.com/equinix-labs/metal-sweeper-action/blob/main/main.go#L79-L98

Reply:
updated